client/my-sites/current-site/index.jsx

```diff
@@ -67,6 +74,36 @@ class CurrentSite extends Component {
 		this.setState( { domainsStore: DomainsStore } );
 	}
 
+	showStaleCartItemsNotice = () => {
+		const { selectedSite } = this.props,
+			cartItems = require( 'lib/cart-values' ).cartItems,
+			staleCartItemNoticeId = 'stale-cart-item-notice';
+
+		// Remove any existing stale cart notice
+		this.props.removeNotice( staleCartItemNoticeId );
+
+		// Don't show on the checkout page?
+		if ( this.props.sectionName === 'upgrades' ) {
+			return null;
+		}
+
+		// Show a notice if there are stale itmes in the cart and it hasn't been shown in the last 10 (cart abandonment)
+		if ( cartItems.hasStaleItem( CartStore.get() ) && this.props.staleCartItemNoticeLastTimeShown < Date.now() - ( 10 * 60 * 1000 ) ) {
+			if ( abtest( 'showCartAbandonmentNotice' ) === 'showNotice' ) {
+				this.props.infoNotice(
+					this.props.translate( 'Your site deserves a boost!' ),
+					{
+						id: staleCartItemNoticeId,
+						isPersistent: false,
+						duration: 10000,
+						button: this.props.translate( 'Complete your purchase' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Complete your paypal purchase' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/global-notice-for-cart-abandonment)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).